addons/website/migrations/13.0.1.0/pre-migration.py

```diff
@@ -35,6 +36,47 @@ def fill_website_rewrite_name(cr):
     )
 
 
+def remove_social_googleplus(env):
+    """On v13, this field has been removed, but it was put on certain places on
+    the website, and they remain unaltered due to the noupdate=1 flag or being
+    a COW (created on write) view, so we directly remove that part from the
+    view if we find the exact HTML expected code that is on the definition.
+    This is done for avoiding side effects, and it means that if you have altered
+    somehow that part, you will need to remove it manually.
+    """
+    for key, code in [
+        (
+            "website.footer_custom",
+            '                                <a t-if="website.social_googleplus" t-att-href="website.social_googleplus"'
```

Comment:
```suggestion
            ' <a t-if="website.social_googleplus" t-att-href="website.social_googleplus"'
```
Just a question, this white space is needed?

Reply:
Yes, it's for letting the view cleaned, removing whole line. At the end, as it has too much perfectly, this space is in the view.

Reply:
If everybody has the same length of white space, then ok.